### PR TITLE
#152592788 Update Navbar and Filters

### PR DESCRIPTION
**Pull request by NaiRobley**
#### What does this PR do?
Update admin navbar and filter components

#### Description of Task to be completed?
As user, **WHEN** I  access the admin page, 
**THEN** I should see the navbar and filters as designed in the new mockup.

#### What are the relevant pivotal tracker stories?
[#152592788](https://www.pivotaltracker.com/story/show/152592788)

#### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/Components/IncidentFilter/IncidentFilter.Component.jsx [hunk 1]
```diff
@@ -33,49 +34,53 @@ export default class IncidentFilter extends Component {
 
 
   render() {
+    const styles = {
```
Comment (Mercy-Muchai) [inline src/Components/IncidentFilter/IncidentFilter.Component.jsx]: We need to move these to the `IncidentFilter.scss` file

File: src/Components/CustomNotification/CustomNotification.Component.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Snackbar from 'material-ui/Snackbar';
+
+// styles
+import './CustomNotification.scss';
+
+const CustomNotification = ({type, open, message }) => {
+  let textStyle = {
```
Comment (jimmykimani) [inline src/Components/CustomNotification/CustomNotification.Component.jsx]: You should place your In-line styles outside the component

File: __tests__/CustomNotification.test.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import { shallow, mount } from 'enzyme';
+import shallowToJSON from 'enzyme-to-json';
+
+import CustomNotification from '../src/Components/CustomNotification/CustomNotification.Component';
+
+describe('CustomNotification component', () => {
+  it('should have all the CustomNotification content', () => {
+    const notification = shallow(<CustomNotification type="success" open={true} message="A test success notification" />);
+    const tree = shallowToJSON(notification);
+    expect(tree).toMatchSnapshot();
```
Comment (Mercy-Muchai) [inline __tests__/CustomNotification.test.jsx]: I don't think we'll be needing any of these snapshot tests now or in future.

File: src/Components/IncidentFilter/IncidentFilter.Component.jsx (hunk 1 above)
Comment (NaiRobley) [reply src/Components/IncidentFilter/IncidentFilter.Component.jsx]: Material-ui components use inline styling, so if the styles were moved to the IncidentFilter.scss they would not apply to the toggle

File: __tests__/CustomNotification.test.jsx (hunk 3 above)
Comment (NaiRobley) [reply __tests__/CustomNotification.test.jsx]: I have made the changes from the snapshots.

**Comment by Mercy-Muchai** [comment]
@NaiRobley We can have the country filter drop-down aligned with the username above. The text-fields for the flag and dates `from` and `to` are too large and the wording should probably be at the centre and not on the far right. We may need to pick either texts between`show incidents` and `filter incidents` versus having both. cc: @andela-hms @Mbarak-Mbigo

**Comment by NaiRobley** [comment]
@Mercy-Muchai I have modified the text fields for `flag`, `from`, `to` and `this week` filters as suggested. I have also aligned the countries filter with the username section above it.

**Comment by Mbarak-Mbigo** [comment]
@NaiRobley Kindly resolve the conflicts 👆

**Comment by NaiRobley** [comment]
@Mbarak-Mbigo resolved.

**Pull request merged**